python/runtime/utils/PyRemoteRESTQPU.cpp

```diff
@@ -89,7 +89,6 @@ class PyRemoteRESTQPU : public cudaq::BaseRemoteRESTQPU {
     pm.addPass(mlir::createCanonicalizerPass());
     pm.addPass(cudaq::opt::createApplyOpSpecializationPass());
     pm.addPass(createInlinerPass());
-    pm.addPass(cudaq::opt::createExpandMeasurementsPass());
```

Comment:
Is this pipeline used by targets other than Braket?

Reply:
Also comes from Pradnya's PR but it shouldn't be used in common pipelnes, we should add it to the yml files if needed instead. Since it works for c++ already I presume we can just remove it here, but I will fix the yml files if the tests fail